### DISCUSSION TITLE
UTF Codes in Windows 10

### DISCUSSION THREAD

**Discussion by tataDan** [wailsapp/wails#1532] [Q&A] [answered]
I have developed an application that retrieves information from a web API and then displays some of that information.  On Linux it correctly displays the flag, but on Windows 10 it displays two characters (e.g. AW) instead.  Here is a screenshot of the partial results from running the API query from a browser: 
![FlagAPI](https://user-images.githubusercontent.com/7750911/177367704-d8eadadb-ce6a-4ec2-a22e-8f14408ef5f8.png).
I suspect that this is a string containing a UTF code.  In fact, this web site (https://www.alt-codes.net/flags) shows the same flag.

Are there any suggestions as to how to resolve this issue?

**Comment by leaanthony** [comment]
Tried meta tags to set encoding?

**Comment by tataDan** [comment, marked as answer]
I believe that I have found an acceptable solution for this issue.  I downloaded the NotoColorEmoji_WindowsCompatible.ttf font from https://github.com/googlefonts/noto-emoji and then copied it to the frontend/src/assets/fonts directory.  I then modified frontend/src/style.css in accordance with the addition of the new font. Lastly, I added a class to the applicable list item in one of the *.svelte files so that the new font would only apply to it and not the entire application.
